PryntTrimmerView/Classes/Parents/AssetVideoScrollView.swift

```diff
@@ -11,117 +11,182 @@ import UIKit
 
 protocol AssetVideoScrollViewDelegate: class {
     func thumbnailFor(_ imageTime: CMTime, completion: @escaping (UIImage?)->())
+    func didUpdateDimensions()
+    func contentOffsetDidChange()
 }
 
-class AssetVideoScrollView: UIScrollView {
+class AssetVideoScrollView: UIView {
 
-    public weak var framesDelegate: AssetVideoScrollViewDelegate?
+    public weak var delegate: AssetVideoScrollViewDelegate?
     
-    private var widthConstraint: NSLayoutConstraint?
-
-    let contentView = UIView()
-    var maxDuration: Double = 15
-    private var thumbnailFrameAspectRatio: CGFloat?
-    private var duration: TimeInterval?
+    let collectionView: UICollectionView
+    let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
+    var maxOnscreenDuration: Double = 1800
+    fileprivate var thumbnailFrameAspectRatio: CGFloat?
+    fileprivate var duration: TimeInterval?
+    fileprivate var thumbnailTimes: [NSValue] = []
+    fileprivate var thumbnailSize: CGSize = CGSize.zero
+    fileprivate var contentWidth: CGFloat = 0
+    fileprivate var lastWidth: CGFloat?
+    fileprivate var lastContentOffset: CGFloat = 0
+    fileprivate var horizontalInset: CGFloat = 15
```

Comment:
all of the above need to be private

Reply:
compiled with swift 3.0 :(

Reply:
Basa. Can you add an issue to the convert this one to latest Swift please?